docs/src/main/paradox/getting-started/running-cluster.md

```diff
@@ -0,0 +1,95 @@
+# Running the Projection in Akka Cluster
+
+Running the Projection with [Akka Cluster](https://doc.akka.io/docs/akka/current/typed/cluster.html) allows us to add two important aspects to our system: availability and scalability.
+A Projection running as a single Actor creates a single point of failure (availability), when the app shuts down for any reason, the projection is no longer running until it's started again.
+A Projection running as a single Actor creates a processing bottleneck (scalability), all messages from the @apidoc[SourceProvider] are processed by a single Actor on a single machine.
+By using a [Sharded Daemon Process](https://doc.akka.io/docs/akka/current/typed/cluster-sharded-daemon-process.html#sharded-daemon-process) with Akka Cluster and [Akka Cluster Sharding](https://doc.akka.io/docs/akka/current/typed/cluster-sharding.html) we can scale up the Projection and make it more available by running as many instances of the same Projection as we have cluster members.
+As akka cluster members join and leave the cluster the Sharded Daemon Process will automatically scale and rebalance Sharded Daemon Processes (Projection instances) accordingly.
+
+Running the Projection as a Sharded Daemon Process requires no changes to our projection handler and repository, we only need to change the way in which the actor that runs the Projection is initialized.
+In the cluster version of this app we use a different configuration that configures akka cluster.
+The main difference in the app itself is that we use @apidoc[ShardedDaemonProcess] to initialize the Projection actor on our behalf.
+Instead of creating single instances of our repository and projection handler we create factory methods that encapsulate their instantiation along with the sharded daemon actors (1 per tag) assigned to this cluster member.
+
+Add a new configuration file `guide-shopping-cart-cluster-app.conf` to your `src/main/resources/` directory.
+This configuration is largely the same as before, but includes extra configuration to enable cluster connectivity and sharding:
+
+@@snip [guide-shopping-cart-cluster-app.conf](/examples/src/test/resources/guide-shopping-cart-cluster-app.conf) { #guideClusterConfig }
+
+Add the `ShoppingCartClusterApp` to your project source:
+
+Scala
+:  @@snip [ShoppingCartApp.scala](/examples/src/test/scala/docs/guide/ShoppingCartApp.scala) { #guideClusterSetup }
+    
+Before running the app we must first run the `EventGeneratorApp` in `cluster` mode in order to generate new shopping cart events for multiple tags, instead of just one.
+Shopping cart events are tagged in a similar way to the sharded entities themselves.
+Given a sequence of tags from `0..n` a hash is generated using the sharding entity key, the shopping cart id.
+The hash is modded `%` by the number of tags in the sequence to choose a tag from the sequence.
```

Comment:
include code snippet to clarify how that tagging looks like?

Reply:
I think there is something for that in the other (reference docs) /running.md

Reply:
I added a mention and link to that specific section in `../running.md`, as well as the source for the `EventGeneratorApp` itself in the previous section of the getting started guide.